include/curl/curl.h

```diff
@@ -1971,6 +1971,10 @@ typedef enum {
   CURLOPT(CURLOPT_PROXY_ISSUERCERT, CURLOPTTYPE_STRINGPOINT, 296),
   CURLOPT(CURLOPT_PROXY_ISSUERCERT_BLOB, CURLOPTTYPE_BLOB, 297),
 
+  /* The CAinfo blob used to validate the peer certificate
+     this option is used only if SSL_VERIFYPEER is true */
+  CURLOPT(CURLOPT_CAINFO_BLOB, CURLOPTTYPE_BLOB, 298),
```

Comment:
Adds a new option, but there's no documentation and no test case verifying it!

Reply:
Good point! Will do if necessary.